docker/nginx/default.conf

```diff
@@ -0,0 +1,33 @@
+upstream origami {
+    server django_app:8000; # for a web port socket (we'll use this first)
+}
+server {
+    listen      80;
+    server_name localhost;
```

Comment:
Since this will be used for the production environment, you can hardcode the URL here.